.prettierrc

```diff
@@ -0,0 +1,32 @@
+{
```

Comment:
Why do we have another file as we already have prettier.config.js which extends from PIW-tools?